docs/alternate_domains.rst

```diff
@@ -1,12 +1,18 @@
 Alternate Domains
 =================
 
+.. note:: This documentation belongs to our community site.
+          If you want to setup an Alternate Domain under `readthedocs.com`_,
+          please read our :ref:`business documentation <business/alternate_domains:Alternate Domains>`.
```

Comment:
We should also rename the "business/" docs. We don't refer to our commercial hosting as "business" anywhere else.